src/app/redux/Item/Item.reducers.ts

```diff
@@ -5,7 +5,7 @@ const initialState = {
     items: [],
 }
 
-function items(state = initialState, action = null) {
+function items(state = initialState, action: { type: string; payload: any }) {
```

Comment:
`items` reducer should have explicit return type.
Check: https://redux.js.org/recipes/usage-with-typescript#type-checking-reducers

`action` should have type for separate interface/type
https://redux.js.org/recipes/usage-with-typescript#type-checking-actions--action-creators

Reply:
Will check on this, sir @iRaySpace